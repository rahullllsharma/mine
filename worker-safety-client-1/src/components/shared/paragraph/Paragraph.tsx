import type { PropsWithClassName } from "@/types/Generic";
import cx from "classnames";

type ParagraphProps = PropsWithClassName<{
  text?: string;
}>;

//TODO Once we start tackling HHDS 2.0, we should revisit this component
export default function Paragraph({
  text = "",
  className,
}: ParagraphProps): JSX.Element {
  return (
    <p className={cx("text-base text-neutral-shade-100", className)}>{text}</p>
  );
}
