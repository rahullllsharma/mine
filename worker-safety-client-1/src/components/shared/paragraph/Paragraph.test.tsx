import { render } from "@testing-library/react";
import Paragraph from "./Paragraph";

describe(Paragraph.name, () => {
  describe("when it renders", () => {
    it("should match the snapshot", () => {
      const { asFragment } = render(<Paragraph text="Lorem ipsum" />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
