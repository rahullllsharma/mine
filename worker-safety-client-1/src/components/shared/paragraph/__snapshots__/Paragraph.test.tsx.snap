// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph when it renders should match the snapshot 1`] = `
<DocumentFragment>
  <p
    class="text-base text-neutral-shade-100"
  >
    Lorem ipsum
  </p>
</DocumentFragment>
`;
